attractions/checkbox/checkbox.svelte

```diff
@@ -1,18 +1,58 @@
 <script>
+  /**
+   * @event {{ value: string; checked: boolean; nativeEvent: Event }} change
+   */
   import { createEventDispatcher } from 'svelte';
   import classes from '../utils/classes.js';
 
   let _class = null;
   export { _class as class };
+  /**
+   * A class string to assign to the `<input>` element.
+   * @type {string}
+   */
   export let inputClass = null;
+  /**
+   * A class string to add to the selector box element.
+   * @type {string}
+   */
   export let selectorClass = null;
+  /**
+   * A CSS style string to assign to the selector box element.
+   * Can be used to make the checkbox represent a color (in conjunction with the `getColorPickerStyles` utility).
+   * @type {string}
+   */
   export let selectorStyle = null;
 
+  /**
+   * Whether the checkbox is checked or not.
+   * @type {boolean}
+   */
   export let checked = false;
+  /**
+   * The value assigned to the checkbox input. Check [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value) for more information.
+   * @type {string}
+   */
   export let value = null;
```

Comment:
Unfortunate inconsistency with the radio button, however, in this case having a single checkbox and not having to pass a value to it could be valuable. But maybe we should cut the tiny convenience and opt for consistency and compliance with standards.

Reply:
I don't get it. Do you mean you want the `value` to be required (not have a default value)?

Reply:
Yeah, I suppose making it required would be the best way to go.